frontend/src/components/common/input/input.tsx

```diff
@@ -37,12 +40,14 @@ const Input: React.FC<Props> = ({
       {hasMultipleRows ? (
         <textarea
           {...field}
+          {...(options as React.TextareaHTMLAttributes<HTMLTextAreaElement>)}
           placeholder={placeholder}
           className={getAllowedClasses(styles.input, styles.textarea)}
         />
       ) : (
         <input
           {...field}
+          {...(options as React.InputHTMLAttributes<HTMLInputElement>)}
```

Comment:
```suggestion
          min={MIN_SHOWNOTE_TIME}
```
what about if I will try to do it with the postman? You also need to change validation